ran-core/src/main/java/io/ran/ClazzMethod.java

```diff
@@ -44,7 +53,7 @@ public <T extends Annotation> T getAnnotation(Class<T> annotation) {
 	}
 
 	public boolean matches(String token) {
-		return this.methodToken.equals(token);
+		return method.toString().equals(token);
```

Comment:
Again a toString, not sure this is 100% stable, through it probably is

Reply:
it used to be the same thing, just the toString() was called in constructor